test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -37,12 +37,29 @@ call_log_manager::call_log_manager(const std::string &call_log_file)
 {
     std::ifstream file(call_log_file);
     if (file.fail()) {
-        std::cout << "File '" << call_log_file << "' either doesn't exist or is not accessible."
+        std::cerr << "File '" << call_log_file << "' either doesn't exist or is not accessible."
                   << std::endl;
         exit(1);
     }
 
-    _call_log = json::parse(file);
+    std::string contents(
+      (std::istreambuf_iterator<char>(file)), (std::istreambuf_iterator<char>()));
+
+    if (contents.empty()) {
+        std::cerr << "File'" << call_log_file << "' is empty." << std::endl;
+        exit(1);
```

Comment:
ditto

Reply:
Updated in [b8e864b](https://github.com/wiredtiger/wiredtiger/pull/8186/commits/b8e864b9d1665b5de62f173504da094f7402a400)